sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/CjkBigramTokenFilter.java

```diff
@@ -58,6 +61,18 @@ public CjkBigramTokenFilter(@JsonProperty(value = "name") String name) {
      * @param ignoreScripts the ignoreScripts value to set.
      * @return the CjkBigramTokenFilter object itself.
      */
+    public CjkBigramTokenFilter setIgnoreScripts(CjkBigramTokenFilterScripts... ignoreScripts) {
+        this.ignoreScripts = Arrays.asList(ignoreScripts);
```

Comment:
This should have a null check. Update this in other places too.

```suggestion
        this.ignoreScripts = ignoreScripts == null ? null : Arrays.asList(ignoreScripts);
```

Reply:
Added checks to all usage of varargs, additionally updated options models as well as they would run into the same issue.